sklearn/utils/class_weight.py

```diff
@@ -48,25 +47,16 @@ def compute_class_weight(class_weight, classes, y):
     if class_weight is None or len(class_weight) == 0:
         # uniform class weights
         weight = np.ones(classes.shape[0], dtype=np.float64, order='C')
-    elif class_weight in ['auto', 'balanced']:
+    elif class_weight == 'balanced':
```

Comment:
There are still two uses of 'auto' in `compute_sample_weight` and 1 more in its documentation.